packages/flutter/lib/src/gestures/events.dart

```diff
@@ -236,6 +237,11 @@ abstract class PointerEvent {
   /// the stylus is flat on that surface).
   final double tilt;
 
+  /// For PointerChange.scroll:
+  ///
+  /// The amount to scroll in the x direction, in logical pixels.
+  final Offset scrollDelta;
```

Comment:
this should probably be in PointerScrollEvent itself, since it doesn't apply to the others.

Also, the documentation is wrong on at least two counts (it's the `y` direction, and actually it's both directions since it's an `Offset` rather than a double).

nit: the first paragraph of any documentation should be a self-contained summary that you can use to describe the member in autocomplete and indexes and other places

Reply:
> this should probably be in PointerScrollEvent itself

I went back and forth on this, and guessed on the side of consistency (nothing had been defined in any subclasses yet). Moved.

> Also, the documentation is wrong on at least two counts (it's the y direction, and actually it's both directions since it's an Offset rather than a double).

Thanks for catching that. I missed this when cleaning up from an earlier version where I had used separate X and Y before realizing that Offset was the standard at this layer.

> nit: the first paragraph of any documentation should be a self-contained summary that you can use to describe the member in autocomplete and indexes and other places

Moot due to moving it. But FYI that format was following the pattern of `orientation` and `tilt` so if that's important from a docs standpoint I can file a bug to clean it up.